test/libsolidity/semanticTests/constants/define_constants_with_external_constant_member_var.sol

```diff
@@ -0,0 +1,25 @@
+==== Source: external.sol ====
+
+library External {
+    uint256 internal constant MAX = 1;
+}
+
+==== Source: usage.sol ====
```

Comment:
This could be a single-file test. I don't think that the fact that these are in separate files affects this in any way.

Reply:
Done